.dockerignore

```diff
@@ -1,4 +1,5 @@
 /*
+!/build*
```

Comment:
im wondering if we can be any more specific - iirc this dir can get large and then be sent to the docker daemon - might not matter so much

Reply:
Would `build_envoy` be OK?

Reply:
i dont rem exactly - my recall is that the `build` folder is used - but perhaps im confusing it (with inside the `build` container) 

If its not used already then just using `build` would be fine.

 If it is - then, yep, using a different folder or pointing to subdir would be best